crates/kornia-imgproc/src/features.rs

```diff
@@ -1,6 +1,22 @@
 use kornia_image::{Image, ImageError};
 
 use rayon::prelude::*;
+use crate::filter::gaussian_blur;
+
+/// Method to calculate gradient for feature response
+pub enum GradsMode {
+    /// Sobel operators
+    SOBEL,
```

Comment:
```suggestion
    Sobel,
```
Enums go in CamelCase in rust